css/css.md

```diff
@@ -76,6 +77,9 @@ The activity for the HTML, CSS, Bootstrap and jQuery lessons is a project that a
 
 [Click here to access the instructions for the CSS portion of the recipe-page project](https://github.com/Techtonica/curriculum/blob/master/projects/recipe-page/phase-2-css-prompt.md). Please don't move on to the prompts for the other topics yet.
 
+While removing *style* tag , you should check for [Inline CSS](https://www.codecademy.com/articles/html-inline-styles) contents from the webpage.
```

Comment:
Why are they checking this? Can you explain more?

Reply:
Yes @alodahl this part is provided as because to remove style tags you need to get the inline css from the webpage, which is essential.

Reply:
Can you explain it for the apprentices with another sentence or 2?

Reply:
also, I hope you mean the style tag and not the style property

Reply:
Yes, I will @alodahl